<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="adminView">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Admin View</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewAllB">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>419</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateB">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>419</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>419</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>419</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logOut">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>489</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchByModelId">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>489</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchID">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>490</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="adminTable">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>100</y>
     <width>501</width>
     <height>291</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>ModelID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>YearMade</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>MeterReading</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
